pacman/model/partitioner_splitters/abstract_splitters/abstract_splitter_common.py

```diff
@@ -66,7 +60,6 @@ def __init__(self, splitter_name=None):
         if splitter_name is None:
             splitter_name = self.DEFAULT_SPLITTER_NAME
         self._splitter_name = splitter_name
-        self._max_atoms_per_core = sys.maxsize
         self._is_fixed_atoms_per_core = False
```

Comment:
Could also get rid of this maybe?

Reply:
(was meant to point at self._is_fixed_atoms_per_core)

Reply:
Yes! but!
yes you are correct.
But also need to strip it out of AbstractSplitterSlice